db/migrate/20220901082732_create_news.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :news do |t|
       t.string :title
       t.string :content
-      t.integer :authorId
+      t.references :user, null: false, foreign_key: true
```

Comment:
in real projects existing migrations cannot be changed. It's okay for the current situation because you can drop DB and create a new one. But in real projects these changes could break DB